Description: Bump version of .so files due to library change
Forwarded: not-needed
Author: Balint Reczey <balint@balintreczey.hu>

Index: b/epan/Makefile.am
===================================================================
--- a/epan/Makefile.am
+++ b/epan/Makefile.am
@@ -54,7 +54,7 @@
 
 noinst_LTLIBRARIES = libwireshark_generated.la libwireshark_asmopt.la
 lib_LTLIBRARIES = libwireshark.la
-libwireshark_la_LDFLAGS = -version-info 0:1:0 -export-symbols libwireshark.sym @LDFLAGS_SHAREDLIB@
+libwireshark_la_LDFLAGS = -version-info 1:0:1 -export-symbols libwireshark.sym @LDFLAGS_SHAREDLIB@
 
 include Makefile.common
 
Index: b/wiretap/Makefile.am
===================================================================
--- a/wiretap/Makefile.am
+++ b/wiretap/Makefile.am
@@ -29,7 +29,7 @@
 
 noinst_LTLIBRARIES = libwiretap_generated.la
 lib_LTLIBRARIES = libwiretap.la
-libwiretap_la_LDFLAGS = -version-info 0:1:0 -export-symbols wtap.sym @LDFLAGS_SHAREDLIB@
+libwiretap_la_LDFLAGS = -version-info 0:2:0 -export-symbols wtap.sym @LDFLAGS_SHAREDLIB@
 
 if HAVE_WARNINGS_AS_ERRORS
 AM_NON_GENERATED_CFLAGS = -Werror

